### DISCUSSION TITLE
Toggle method for belongsToMany relationship

### DISCUSSION THREAD

**Discussion by Joandres-Lara** [sequelize/sequelize#14503] [Help & Questions]
Hello there, do you think a `toggle` method is necessary to insert or remove identifiers from many-to-many relationships? I use the Laravel ORM as an example, which contains a helper to make this association.

https://laravel.com/docs/9.x/eloquent-relationships#toggling-associations

Thanks very much for feedback.

**Comment by ephys** [comment]
I'm ok with this feature being added, but I think we will wait until https://github.com/sequelize/sequelize/issues/14302 is implemented first, to avoid further increasing the typescript boilerplate.

Feel free to open a feature request for it :) https://github.com/sequelize/sequelize/issues/new?assignees=&labels=&template=feature_request.md&title= (and/or opening a PR that implements it, but that would have to be after #14302)

**Reply by Joandres-Lara** [reply]
Great, do you think this feature is part of the good-first-issue tags? I'll wait until the implementation of #14302, to propose a fix for this feature.

Regards. @ephys

**Reply by ephys** [reply]
For sure, `good-first-issue` could be applied to it!
